### PR TITLE
add text-iniput labels and disabled states for certain buttons

### PR DESCRIPTION
**Pull request by NotNestor**
Add json for general labels that are used throughout the app
Add text input json (code-input and search should probably be moved to this file in my opinion)
Add missing disabled states to buttons

### CODE REVIEW

File: components/button.json [hunk 1]
```diff
@@ -180,7 +202,8 @@
     },
     "hyperlink": {
       "#normal": {
-        "text": "@theme-text-accent-normal"
+        "text": "@theme-text-accent-normal",
+        "border": "@theme-background-ghost"
       },
```
Comment (NotNestor) [inline components/button.json:207]: add a transparent border for normal theme. desktop needs normal state in validator, just have focused will cause the build to fail

File: components/text-input.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{ 
+  "textInput": {
```
Comment (paulwitty) [inline components/text-input.json:2]: I had a debate with Colin on my team about this - can we use "textinput" for the component name please (i.e. neither camelcase nor kebab-case)

Comment (NotNestor) [reply components/text-input.json:2]: sure i'll update

File: components/button.json (hunk 1 above)
Comment (NotNestor) [reply components/button.json]: removed border for now until we decide how this should be handled

**Pull request merged**